test/conformance/api/v1/revision_timeout_test.go

```diff
@@ -99,6 +104,7 @@ func TestRevisionTimeout(t *testing.T) {
 		expectedStatus: http.StatusOK,
 		sleep:          15 * time.Second,
 		initialSleep:   0,
+		expectedBody:   "activator request timeout",
```

Comment:
We should delete the test like @izabelacg recommended since it was originally intended to test idle timeout and we have that in our e2e test

https://github.com/knative/serving/issues/15089#issuecomment-2091836604

We now know why this test was passing when it shouldn't have.

Reply:
Once you write a 200 back you cant fail it. I dont think we should avoid testing this.

Reply:
conformance tests can't assume things about the implementation. This isn't defined in the knative spec so I'd rather remove it from these tests.

I believe we have the coverage in our e2e tests

Reply:
As we discussed it is not in the spec but imho it should be defined as it is something the user sees from our implementation (user gets a body txt that is part of the system behavior). Who should we ask about Cloud run?